Lib/random.py

```diff
@@ -261,6 +261,8 @@ def randint(self, a, b):
     def _randbelow_with_getrandbits(self, n):
         "Return a random int in the range [0,n).  Raises ValueError if n==0."
 
+        if n == 0:
+            raise ValueError("Boundary cannot be zero")
         getrandbits = self.getrandbits
         k = n.bit_length()  # don't use (n-1) here because n can be 1
```

Comment:
We can use `(n-1)` now.
```suggestion
        k = (n-1).bit_length()
```

Reply:
I think we want to ensure that the values produced for a given seed don't change, so we probably can't change this anymore.